spectral_fitting/fitting_with_spectralcube.ipynb

```diff
@@ -0,0 +1,697 @@
C+agYikYjzHshDwFTp8TQwUHq8Bng5sA14AfAx4BfKP2iMGQVuiiVKidygOw0NXGMQKNYMlAxEohBnMphiIQEMAMGvegz4nrV2HBg3xmys/KC1dpRiU9I8Y4wfWaQSqWIH8vqGP+eWZi51IohJpNPFmQweBHYC9wNXUmwWAjgAnGeMGQTWAidjjEkSMOQ0di+DQJAMUhHEJNLp4uwz2ANcYoy5H5gDeowxb7XWzgDvodifsAf4oxhjkgQ0epezQHFoqaajEIlCbDUDa20OuLbi6X2l1+4A7ogrFknWkDuNkx5q+HNBzcD31UIoEjZddCaxG3JPQYPzEkHpngaFHBRmIohKpLMpGUjshhu8/3FgfmI7NRWJhE7JQGLl+15xorqmagbFz/iakkIkdEoGEq/cBK7jN1UzcLr6wUlBTlchi4RNyUDiFZzIm6gZOI5TrB2U1QzGMjnGMrmwohPpWEoGEis/mFso3fjQUig2FfmlPoOxbJ71o3exfvQuJQSRFikZSLxy40x5/Thuc5eOuT1D8zWDbL5AwfMpeD7ZvBdmlCIdR8lAYuXPjjPhrWr688WagTqQRcKmZCDxyo0z4Q2s/L4a3J5hyGtoqUjYlAwkVn5ugsmWawYaTSQSNiUDiVerzUTdw6Ab3IiETslAYuXnxpnwW2gm6h1Rn4FIBJQMJF6z4y01E6V618DsWIgBiQiskAyMMd01nl9yAxqRevi5sdY6kHtH5pPB5MxcWGGJdLyVagZfNMaky58wxvwC8J3oQpIzWq7FPoPeNZAbI+U6XHXL/SEGJtLZVrqfwTeBfzTG/BLFxPERijey/7moA5Mzkz87zmQLNYNU7xr82XH2v/0KLt59b4iRiXS2ZWsG1tqbgW8A/5fi7SqngUuttftjiE3ORC3XDEYAn0E3E15MIrJyB7K19mPAVyne0P6PrbX5yKOSM5Lve5CbaPGiszXFB+pEFgnVss1Expj/A/iAA1wA7DXGPAFgrf3N6MOTM4mfmwLfY9Jv5aKzIXBcfCUDkVCt1Gfw+RqPRRpWmCmNAmqhmchxXNyeEdUMREK2bDKw1u6LKxA583mzrScDgFTfmlLNYCSEqEQEdNGZxMibHYeuAeZWrJAuz+3RhWciYVMykNh4s+NN3eGsktu7Bj+nZCASJiUDiY03M97UvY8rub0jMHMyhIhEJNBafb0Bxpgu4FZgE/CAtfb6ite7gceBl1lrT8UVl8THmx0LpWaQ6l2DP3UshIhEJBBnzeD1wCPW2u3AamPMKyte/wPgrBjjkZgVZsbCqxmoz0AkVLHVDIDLgC+UHt9DcVqL7wAYY9YBrwT+X7UPGmNGgZuiD1GiFGqfgZKBSKjiTAZDFK9ihuK0FuWXod4E/Anw59U+aK0dBUbLnzPG+KFHKJHyZsZwute3vJyUagYioYuzmWiKhQQwAEwAGGMuAtLW2kdijEUS4M2OQxjNRD2qGYiELc5k8CCws/T4SuCB0uOrgZcbY/YClwB/HWNMEiNvZgwnpGYicmM4eCFEJSIQbzLYA1xijLkfmAN6jDFvtdb+ubX2UmvtTuAh4E0xxiQxKsyOhVIzSPWtAd9jlZMNISoRgRj7DKy1OeDaiqf3VbxnZ1zxSPy87AlSvWuhxRK921ucuXS1e4pThdamthCRIl10JrHwvbnSdQZrW16W2z0MOIy4Uyu+V0Tqo2QgsfBKVww7va0nA8dNQfcww66uTRQJi5KBxKKQPV58EEIyAKBnhNVKBiKhUTKQWBSyJ3C6+nC6+kNZntMzopqBSIiUDCQW3sxx3L514S1QNQORUCkZSCwK8yOJwlGsGagDWSQsSgYSCy8bcs2gWzUDkTApGUgsCtnjpEJMBk7PCKtVMxAJjZKBxCL0ZNB7FmtTk6EtT6TTKRlILLzsCdy+8PoM6FvPWncivOWJdDglA4lFYeY4qd51TM7mQ1le74CSgUiYlAwkFl7mONnUMNt27yPlOvSlWzv0htdsot+dZf8f/GRIEYp0NiUDiZzv+xQyR8inz6Lg+ezftYOR/u6WlpnqK94kZ7UzHkaIIh1PyUAi5+cm8QszUDqBD/WkW16m2zsCbhd+9ljLyxIRJQOJQSFzBACnr/VbXgYcxyXVd5aSgUhIlAwkcuMnn4VUD6SHQl1uqu8smFEyEAmDkoFEaiyT4zf/6h6enR1ifGYu1GWn+jfgKxmIhELJQCKVzXusdcY47q0mmy+Eumy3/yzIHg11mSKdSslAIrcuNc6xQuv3Pq6U6l+vmoFISJQMJHLr3AlOFIZDX26qfwO+agYioVAykMitS41xzAu/ZtA1cDZMHwp9uSKdSMlAIrcuNcHxKJqJVp2Dn3ke8ENftkinUTKQyK13o6oZbIJClkEnE/qyRTqNkoFEyvcKnJUa43AhxBlLS1IDmwDYmDoR+rJFOk1XXCsyxnQBtwKbgAestdeXvXYD8DqK9f3rrLXfjSsuiVj2MF2Ox/NzId7lrMRNr4LuYTakToa+bJFOE2fN4PXAI9ba7cBqY8wrAYwxG4CfsdZeDrwReG+MMUnE/OnnyPspjnvhjyYCcPrPVjIQCUFsNQPgMuALpcf3AFcA3wFOAP+lLJ5c5QeNMaPATdGHKGHzp5/jaGEEj1Q0K+g/m40hJIOxTPGwa3U2VZHTVZzJYAgIblo7DQwAWGvngJPGmD7gFuDGyg9aa0eB0fLnjDEaQnIa8DPPRdJfEHD6N7E+dbylZYxlcqwfvQuAo6PXKCFIR4qzmWiKUgIo/T9/mypjzADwZeAvrLXfjjEmidr0czxfCL+/IBBGM1E271HwfAqeTzbvhRSZyOklzmTwILCz9PhK4IGy1/YAn7bW3hZjPBIDf/oQz0dZMxjYzDkpTUkh0qo4m4n2AH9jjLkfeBjoMca8FdgPbAf6jTHXAdZa++YY45II+dPPcrjw0siW7wycy+auo/i+Wg1FWhFbMrDW5oBrK57eV/p/MK44JF7+9HMcmtsZ2fKdwXMZdDMwex
```

Comment:
Not sure if there will ever really be any benefit to do so?

Reply:
Probably not in the near future. It would have to be a cube/image with a huge spatial area. I added this here just to match the spectral fitting w/ dask above.

Reply:
Because I cleared output, this was referring to the mention that we don't have 2D Projections implemented in dask form.